concepts/Code_status/carePlan_getItemValues.Rmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "carePlan_getItemValues"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Connect to database.
+
+```{r}
+require("RPostgreSQL")
+drv <- dbDriver("PostgreSQL")
+con <- dbConnect(drv, dbname = "eicu",
+                 host = "localhost", port = 5432,
+                 user = "josephpark", password = rstudioapi::askForPassword("Database password"),
+                 options = "--search_path=eicu")
```

Comment:
For most people, I think the schema will be something like `eicu_crd`

Reply:
With the configuration setting, hopefully it is clearer despite the different schema name.